test/typescript/examples.test.tsx

```diff
@@ -22,6 +22,7 @@ function MyComponent() {
 // page uses the hook
 function Page() {
   const [t, i18n] = useTranslation();
+  // const { t, i18n } = useTranslation();
```

Comment:
please remove comment

Reply:
Actually, we need both if you want to merge this PR.  If you are expecting it to return both.

Reply:
I'll push some test changes.